cosmos-file-list.sh

```diff
@@ -0,0 +1,11 @@
+# /bin/bash
+
+ls -1 $(grep -slR 'message Msg' ./cosmos-sdk/proto | egrep '\.proto$') \
+    $(egrep -slR 'enum|message' ./cosmos-sdk/third_party/proto/tendermint/ | egrep '\.proto$') \
+    $(egrep -slR 'enum|message' ./cosmos-sdk/third_party/proto/google/ | egrep '\.proto$') \
+    ./cosmos-sdk/proto/cosmos/bank/v1beta1/bank.proto \
+    ./cosmos-sdk/proto/cosmos/base/v1beta1/coin.proto \
+    ./cosmos-sdk/proto/cosmos/gov/v1beta1/gov.proto \
+    ./cosmos-sdk/proto/cosmos/staking/v1beta1/staking.proto \
+    ./cosmos-sdk/third_party/proto/confio/proofs.proto \
+    | sort | uniq
```

Comment:
Maybe we should keep this list static (since we need to list some files manually anyway).

Reply:
I think we should leave it like this to show how the list is made. With [GRAPH-384](https://figmentio.atlassian.net/browse/GRAPH-384) we won't need to list file manually anymore.